pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/JdbiFormSectionBlock.sql.stg

```diff
@@ -2,6 +2,7 @@ group JdbiFormSectionBlock;
 
 queryOrderedFormBlockDtosBySectionIdsAndInstanceGuid() ::= <<
 select fsb.form_section_id,
+       null as parent_block_id,
```

Comment:
Same here.

Reply:
A regular block doesn't have a parent block id since it's not nested, so yes.